src/main/java/Aether.java

```diff
@@ -0,0 +1,206 @@
+import java.util.Scanner;
+
+public class Aether {
+    private boolean isExit = false;
+    private static final int MAX_TASKS = 100;
+    private Task[] tasks = new Task[MAX_TASKS];
```

Comment:
plural form of the class name - 'tasks'- enhances readability by giving a clear picture of what is being stored. Keep this up!